src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml

```diff
@@ -33,6 +33,7 @@
     </StringProperty.DataSource>
   </StringProperty>
 
+  <BoolProperty Name="RunCodeAnalysis" DisplayName="Run Code Analysis" Visible="False" />
```

Comment:
I know you're being consistent, but non-visible properties don't need to have DisplayName.

Reply:
I'll come through and clean all of these up eventually anyway.